middleware.test.ts

```
@@ -0,0 +1,396 @@
+      await expect(updateSession(mockRequest)).rejects.toThrow('Supabase client creation failed')
+    })
+
+    test('should handle auth.getUser() errors', async () => {
+      mockSupabaseClient.auth.getUser.mockRejectedValue(new Error('Auth error'))
+
+      await expect(updateSession(mockRequest)).rejects.toThrow('Auth error')
+    })
+
+    test('should handle missing environment variables', async () => {
+      const originalEnv = process.env
+      delete process.env.NEXT_PUBLIC_SUPABASE_URL
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)